recipes/mailio/all/conanfile.py

```diff
@@ -70,9 +70,20 @@ def validate(self):
         except KeyError:
             self.output.warn(f"{self.ref} has no support for the current compiler. Please consider adding it.")
 
+    def _cmake_new_enough(self, required_version):
+        try:
+            import re
+            from io import StringIO
+            output = StringIO()
+            self.run("cmake --version", output)
+            m = re.search(r"cmake version (\d+\.\d+\.\d+)", output.getvalue())
+            return Version(m.group(1)) >= required_version
+        except:
+            return False
+
     def build_requirements(self):
         # mailio requires cmake >= 3.16.3
-        self.tool_requires("cmake/3.25.0")
+        self.tool_requires("cmake/[>=3.16.3 <4]")
```

Comment:
Using this version range, can we remove the `_cmake_new_enough` function?

Reply:
Good catch!

Reply:
There is nothing to fix in this recipe. This PR can be closed since another PR has already introduced version range.

Reply:
Yup, just saw that. Thanks for the heads up :)